CONTRIBUTING.md

```diff
@@ -19,7 +19,6 @@ Be careful to observe the script's output, it may have errors or ask you to do a
 Once in Xcode, select the `FluentPostgreSQL-Package` scheme and use `CMD+U` to run the tests.
 
 When adding new tests (please do 😁), don't forget to add the method name to the `allTests` array. 
```

Comment:
You can delete this line as well since `allTests` is no longer a thing. Could you change it to
> Please add tests to cover bug fixes and new features.

Reply:
Sure, will do.

I just realise it's **`allTest` "array"** actually, I thought the sentence meant adding the new test to `testAll()` function in `FluentPostgresDriverTests`.

By the way, aren't the tests (`testAggregate()`... `testUnique()`) executed twice, once called from `testAll()` [1] and then individually[2]? Is this for convenience? A bit of redundancy although tests run fairly fast?


Refs:

[1]:
```
public final class FluentBenchmarker {

    public func testAll() throws {
        try self.testAggregate()
        try self.testArray()
        try self.testBatch()
        ...
        try self.testTransaction()
        try self.testUnique()
    }
```

[2]
```
final class FluentPostgresDriverTests: XCTestCase {
    func testAll() throws { try self.benchmarker.testAll() }
    func testAggregate() throws { try self.benchmarker.testAggregate() }
    func testArray() throws { try self.benchmarker.testArray() }
    ...
    func testTransaction() throws { try self.benchmarker.testTransaction() }
    func testUnique() throws { try self.benchmarker.testUnique() }
```